spec/formatting.md

```diff
@@ -428,6 +432,32 @@ These are divided into the following categories:
     > when * {The value is not one.}
     > ```
 
+  - **Variable Redefinition errors** occur when
+    the left-hand side of a variable declaration is already defined,
+    either by a previous local variable declaration,
+    or by an external definition.
+
+    > For example, attempting to format the following message
+    > must result in a Variable Redefinition error,
+    > because there is another declaration for `var`
+    > that textually precedes
+    > the declaration of `var` to be `{|horse|}`.
+    >
+    > ```
+    > let $var = {|cart|}
+    > let $var = {|horse|}
+    > {The value is {$var}.}
+    > ```
+    >
+    > Attempting to format the following message in a context
+    > where `var` is externally defined
+    > should also result in a Variable Redefinition error.
```

Comment:
see above. And "should" is super wrong 😉 

Just say "results" or "also results"